### ISSUE TITLE
No pipelines triggered when changes are made in quick start app code and committed using Che.

### ISSUE THREAD

**Issue by Preeticp** [openshiftio/openshift.io#4756]
##### Issue Overview

After the quickstart codebase is edited in Che and the change successfully committed, pipelines should be triggered in CRT, but this does not happen. I tried it with Nodejs-health-check booster and mission.

##### Expected Behaviour
As per the happy path/getting started path in CRT, an user uses the wizard to launch a Nodejs app, sees that pipelines are created, promotes his app from stage to run and opens his Che workspace to edit the code. Once he edits the code in Che and commits it to git, the pipelines should be triggered again.
##### Current Behaviour
No pipelines triggered when changes are made in quick start app code and committed using Che.
##### Steps To Reproduce

1. Launch a node js app
2. Check your pipelines and promote to run environment
3. Open your che workspace, change code and commit, commit mssg is successful, you can see your changes in git but no pipelines are triggered in CRT

##### Additional Information

![screenshot from 2019-02-12 15-11-32](https://user-images.githubusercontent.com/16647625/52632005-4f577c00-2ee6-11e9-9f73-b665888a8285.png)
![screenshot from 2019-02-12 15-18-59](https://user-images.githubusercontent.com/16647625/52632004-4f577c00-2ee6-11e9-86a4-df8f15275ef8.png)
![screenshot from 2019-02-12 15-25-21](https://user-images.githubusercontent.com/16647625/52632003-4ebee580-2ee6-11e9-9e81-54faa88f2d4f.png)

**Comment by hrishin** [comment]
@Preeticp how did git push after commit as well right?

**Comment by Preeticp** [comment]
Yes, the first shot says it has been pushed to origin, the second shot shows the changes in the code.

**Comment by Preeticp** [comment]
Following instructions from here: https://docs.openshift.io/getting-started-guide.html#committing_pushing_changes_git-hello-world

**Comment by piyush-garg** [comment]
@Preeticp @hrishin I tried to reproduce this,  for me, builds are getting triggered on webhook event if Jenkins is running, In case if Jenkins is idled, then on webhook events, the proxy receives the request and return 202 in response, wake up Jenkins but do not trigger build later.

cc @sthaha
